app/services/bulk_upload/sales/year2024/row_parser.rb

```diff
@@ -862,6 +852,7 @@ def attributes_for_log
     attributes["pregla"] = field_68
     attributes["pregghb"] = field_69
     attributes["pregother"] = field_67
+    attributes["pregblank"] = no_buyer_organisation
```

Comment:
does this exist as an option on the BU template/ does it have it's own field?

Reply:
Similarly, my understanding is that this isn't being introduced until 2025, per comment on https://dluhcdigital.atlassian.net/browse/CLDC-3245